snap/snapcraft.yaml

```diff
@@ -0,0 +1,42 @@
+name: geewallet # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.1' # just for humans, typically '1.2+git' or '1.3.2'
+summary: minimalistic and pragmatist crossplatform lightweight opensource brainwallet # 79 char long summary
```

Comment:
only 79 is allowed? damn, I guess adding "multicurrency" word is more important than "minimalistic" or "pragmatist"; which word would you take out?

Reply:
Not sure about this.

Reply:
yes tell me which word you dislike the most: "minimalistic" or "pragmatist"?

Reply:
I don't dislike either of them.